docs/content/guides/developer/getting-started/sui-install.mdx

```diff
@@ -29,28 +29,91 @@ brew install mystenlabs/tap/sui
+    - `sui-node-ubuntu-x86_64`: Run a local node.
+    - `sui-test-validator-ubuntu-x86_64`: Run test validators on a local network for development.
+    - `sui-tool-ubuntu-x86_64`: Provides utilities for Sui.
 
     The Sui binary takes several minutes to download and install files, so make sure you allocate enough time for it to complete. Actual time to complete depends on your network connection and computer specifications.
 
+</TabItem>
+
+<TabItem value="mac" label="macOS">
+
+1. Go to https://github.com/MystenLabs/sui.
+1. In the right pane, find the **Releases** section.
+
+    ![Sui releases in GitHub](./images/releases.png)
+1. Click the release tagged **Latest** to open the release's page.
+1. In the **Assets** section of the release, select the .tgz compressed file that corresponds to your operating system.
+1. Double-click the downloaded file. If the file doesn't automatically expand, manually unzip the file.
+1. Open the expanded folder. Navigate to the `target/release` directory and double-click the appropriate binary to install, beginning with `sui-macos-<ARCHITECTURE>`:
+
+    - `sui-data-ingestion-macos-<ARCHITECTURE>`: Massage Full node data for indexer to store in a database.
+    - `sui-faucet-macos-<ARCHITECTURE>`: Local faucet to mint coins on local network.
+    - `sui-indexer-macos-<ARCHITECTURE>`: An indexer for a local Sui network.
+    - `sui-macos-<ARCHITECTURE>`: Main Sui binary.
+    - `sui-node-macos-<ARCHITECTURE>`: Run a local node.
+    - `sui-test-validator-macos-<ARCHITECTURE>`: Run test validators on a local network for development.
+    - `sui-tool-macos-<ARCHITECTURE>`: Provides utilities for Sui.
+
+    The Sui binary takes several minutes to download and install files, so make sure you allocate enough time for it to complete. Actual time to complete depends on your network connection and computer specifications.
+
+</TabItem>
+
+<TabItem value="win" label="Windows">
+
+1. Go to https://github.com/MystenLabs/sui.
+1. In the right pane, find the **Releases** section.
+
+    ![Sui releases in GitHub](./images/releases.png)
+1. Click the release tagged **Latest** to open the release's page.
+1. In the **Assets** section of the release, select the .tgz compressed file that corresponds to your operating system.
+1. Extract all files from the .tgz file into the preferred location on your system. These instructions assume you extract the files into a `sui` folder at the root of your C drive for demonstration purposes. Replace references to this location in subsequent steps if you choose a different directory.
+1. Navigate to the `target\release` directory of the expanded folder. Rename each file to remove the `-windows-x86_64` suffix from each file name. For example, rename `sui-windows-x86_64.exe` to `sui.exe`. If you don't do this step, you must type `sui-windows-x86_64` to execute commands instead of simply `sui`. When complete, you should have the following .exe files:
```

Comment:
I think you'd need to do this on all platforms, not just Windows. All our other documentation assumes these binaries will be named "sui" not "sui-platform-arch", as does I think some of the VSCode support.

Reply:
Will test.